pages/k8s/1.19/release-notes.md

```diff
@@ -76,6 +76,15 @@ the new [SR-IOV CNI documentation][cni-sriov].
 For a full list of the changes introduced in Kubernetes 1.19, please see the
 [upstream release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
 
+- MetalLB Operator
```

Comment:
Let's move this up, right after the SR-IOV paragraph ("For a full list of the changes..." should come last).